### PR TITLE
Arbiter of the Ideal / DigEffect update

### LINKED ISSUE #1670: Arbiter of the Ideal
The card it returns needs to have the counter and enchantment type already when it ETB

### PR DESCRIPTION
**Pull request by Northmoc**
closes #1670

### CODE REVIEW

File: forge-game/src/main/java/forge/game/ability/effects/DigEffect.java [hunk 1]
```diff
@@ -421,6 +421,15 @@ else if (!sa.hasParam("NoLooking")) {
                                         sa.getParamOrDefault("WithCounterNum", "1"), sa);
                                 c.addEtbCounter(CounterType.getType(sa.getParam("WithCounter")), numCtr, player);
                             }
+                            if (sa.hasParam("AddTypes")) {
```
Comment (tool4ever) [inline forge-game/src/main/java/forge/game/ability/effects/DigEffect.java:424]: @Hanmac
Should this be done like `AnimateSubAbility$` instead and prefill LKI?

Comment (Hanmac) [reply forge-game/src/main/java/forge/game/ability/effects/DigEffect.java:424]: AnimateSubAbility would be more flexible

**Review by tool4ever — APPROVED** [review]
Thanks!

**Review by Hanmac — APPROVED** [review]
👍

**Pull request merged**